src/kubectl.ts

```diff
@@ -5,7 +5,7 @@ import { FS } from './fs';
 import { Shell, ShellHandler, ShellResult } from './shell';
 import * as binutil from './binutil';
 import { Errorable } from './errorable';
-import { parseLineOutput } from './kubectlUtils';
+import { parseLineOutput, POD_OUTPUT_PARSING_REGEX } from './kubectlUtils';
```

Comment:
The regex appears to be used only in `kubectl`, not in `kubectlUtils` - why not declare it here, rather than declaring it there and having to import?

Reply:
Fixed